zkevm-circuits/src/state_circuit/constraint_builder.rs

```diff
@@ -28,9 +28,14 @@ pub struct Queries<F: Field> {
     pub field_tag: Expression<F>,
     pub storage_key: RlcQueries<F, N_BYTES_WORD>,
     pub value: Expression<F>,
+    pub prev_value: Expression<F>,
+    pub value_prev: Expression<F>,
```

Comment:
I remember I was trying to come out with some other names for `prev_value` and `value_prev` in the first version of the state circuit, but didn't have some good idea - but perhaps just add some comment what's the difference between the two?

Reply:
This naming conundrum was the main reason I ended up getting rid of the prev_value column :)